pom.xml

```diff
@@ -51,6 +51,7 @@
         <online.repo.version>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</online.repo.version>        
 
         <ohc.version>2.5.0-SNAPSHOT</ohc.version>
+        <ohw.version>2.5.0-SNAPSHOT</ohw.version>
```

Comment:
I don't think we need to add a specific version - using `project.version` should be just fine (that's what we do for other addons as well).